src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discountFor3Days = 20;
+  const discountFor7Days = 50;
+
+  if (days >= 7) {
+    return days * price - discountFor7Days;
+  }
+
+  if (days >= 3) {
+    return days * price - discountFor3Days;
+  }
+
+  return days * price;
```

Comment:
I'd recommend moving `days * price` to a var so that not to repeat yourself 3 times